docker/README.md

```diff
@@ -0,0 +1,15 @@
+# Ubermag OOMMF container
+
+The files in this directory create the Docker image containing the OOMMF
+installation, which is used by [Ubermag](https://ubermag.github.io) if Ubermag
+uses Docker to execute oommf.
+
+This can be ignored by most users.
+
+We have a workflow that creates the container periodically:
+
+[![ubermag-container](https://github.com/fangohr/oommf/actions/workflows/ubermag-container.yml/badge.svg)](https://github.com/fangohr/oommf/actions/workflows/ubermag-container.yml)
+
+Pushing the image to dockerhub (`make push`) is a manual procedure.
```

Comment:
- If publishing is mentioned, we could outline all steps required for creation and publication.
- Publishing directly from the CI should also be possible (in case there is any advantage to do so). Does it make any difference whether the underlying Ubuntu is updated regularly?

Reply:
If we can automate updating the docker hub image for Ubermag, then that's probably good. Have you got time to look into it?

Should we do it whenever there is a new release on this repo?

Reply:
There is an example on how to publish to docker hub here: https://docs.github.com/en/actions/guides/publishing-docker-images#publishing-images-to-docker-hub
Not sure if we would rather follow this or try to rely on our Makefile as much as possible. The latter might need some changes to be compatible with secrets stored on GitHub (and made available via environment variables).

Automatically updating the image on releases on this repo sounds good.